src/process/init.rs

```diff
@@ -188,6 +188,16 @@ pub fn container_init(args: ContainerInitArgs) -> Result<()> {
         command
             .pivot_rootfs(rootfs)
             .with_context(|| format!("Failed to pivot root to {:?}", rootfs))?;
+
+        if let Some(sysctl) = &linux.sysctl {
+            let sys = PathBuf::from("/proc/sys");
+            for kernel_param in sysctl {
+                let path = sys.join(kernel_param.0.replace(".", "/"));
+                fs::write(path, kernel_param.1.as_bytes()).with_context(|| {
+                    format!("failed to set sysctl {}={}", kernel_param.0, kernel_param.1)
+                })?;
```

Comment:
How about this code?
```suggestion
            for (kernel_param, value) in sysctl {
                let path = sys.join(kernel_param.replace(".", "/"));
                log::debug!(
                    "apply value {} to kernel parameter {}.",
                    value,
                    kernel_param
                );
                fs::write(path, value.as_bytes())
                    .with_context(|| format!("failed to set sysctl {}={}", kernel_param, value))?;
            }
```